php-checks/pom.xml

```diff
@@ -33,6 +33,10 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+    </dependency>
```

Comment:
I'm not sure if adding JUnit5 to this ticket was a good idea. The number of executed tests differ on CI, please compare build output https://cirrus-ci.com/task/5529826455453696 (master, executed tests: 54, 429, 137) and https://cirrus-ci.com/task/6002742234185728 (this branch, 161, 3, 5). I'm curious why the build is green again. 🤔 

As a quick fix please add `junit-vintage-engine` where needed.

Reply:
Good suggestions! I will add the vintage-engine so that we have all tests again. As I did add Junit 5 to some tests, I don't feel I should do the work of removing it and doing it again for another ticket, just to have it in separate PR's.

Reply:
#Tests executed are 943, 429 and 137